amundsen_application/api/v0.py

```diff
@@ -13,7 +13,7 @@
 
 LOGGER = logging.getLogger(__name__)
 
-blueprint = Blueprint('api', __name__, url_prefix='/api')
+blueprint = Blueprint('main', __name__, url_prefix='/api')
 
```

Comment:
This was needed as flask_restful creates API endpoints as 'api' and this being named api collided with the flask_restful contents. From testing, this doesn't appear to affect anything.